frontend/src/components/CategorySelect/CategorySelect.scss

```
@@ -0,0 +1,26 @@
+.category_select {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--space-vs) 0;
+
+  .category_select__item {
+    padding: var(--space-s) var(--space-m);
+    border-radius: var(--border-radius-m);
+    border: 1px solid rgba(0, 0, 0, 0);
```

как я понимаю идея в прозрачности. 

ты можешь задать прозрачность используя значение `transparent`

 ```suggestion
    border: 1px solid transparent;
```